### DISCUSSION TITLE
wire:ignore not ignoring when update happened  on an other component

### DISCUSSION THREAD

**Discussion by mansoorkochy** [livewire/livewire#4908] [1. Help]
I have two components, one rests as a child of another one, I want to ignore an element in the parent and it works well for parent updates ignoring but losing affection for child updates, the behavior is a pretty strang mirror to livewire's behavior.

`<parent-component> <--! event at the level ignore the element-->`
     `<livewire-child></livewire-child> <--! event at this level not ignoring-->`
    `<div wire:ignore></div>`
`</parent-component>`

**Comment by denizozturk** [comment]
The "self" modifier restricts updates to the element itself but allows modifications to its children.
```
wire:ignore.self
```
As an example:
```
<div id="parent-element" wire:ignore.self>
    <livewire:components.button />
</div>
```

**Reply by mansoorkochy** [reply]
Sure I have used the `self` modifier but it is also not working

**Reply by denizozturk** [reply]
> Sure I have used the `self` modifier but it is also not working

Can you share your Blade code sample? I want to see how you use this

**Comment by mansoorkochy** [comment]
Here my simple blade html
`<div class="row postBarWrapper" style="background-color: lightgray">


        @foreach($posts as $index => $eachPost)
            <div class="col-lg-12 " wire:key="{{ $eachPost['uuid'] }}">

                @livewire('partials.livewire-post' ,[
                'post' => $eachPost,
                'options' => $options,
                'relatedCommentOnly' => $relatedCommentOnly,
                'post_uuid' => $eachPost['uuid'],
                ],key(time() . $index))
            </div>
        @endforeach
        @if($options['section'] != 'single-post')
            <div wire:ignore.self class="col-lg-12" id="postsBottom" style="padding: 0px; margin-top: 7px">
               <div class="ph-item" style="margin-bottom: 10px;">
                <div class="ph-col-12 " style="">
                    <div class="ph-row">
                        <div class=" big" style="width: 60px; height: 60px;  border-radius: 30px"></div>
                        <div class="big" style="margin-left: 10px"></div>
                        <div class="ph-col-6 big"  style="margin-top: 30px"></div>
                        <div class="ph-col-2 empty big"></div>
                        <div class="ph-col-4"></div>
                        <div class="ph-col-8 empty"></div>
                        <div class="ph-col-6"></div>
                        <div class="ph-col-6 empty"></div>
                        <div class="ph-col-12"></div>
                    </div>
                    <div class="ph-picture"></div>
                    <div class="ph-row">
                        <div class="ph-col-2 big"></div>
                        <div class="ph-col-3 empty big"></div>
                        <div class="ph-col-2 big"></div>
                        <div class="ph-col-3 empty big"></div>
                        <div class="ph-col-2 big"></div>
                    </div>
                    <div class="ph-row" style="margin-bottom: 20px">
                        <div class=" big" style="width: 40px; height: 40px;  border-radius: 30px"></div>
                        <div class=" empty big" style="margin-left: 10px"></div>
                        <div class="ph-col-8 big"  style="margin-top: 15px; height: 30px"></div>
                        <div class="empty big" style="margin-left: 10px"></div>
                        <div class="ph-col-2 big" style="height: 30px;margin-top: 15px;"></div>
                    </div>
                </div>
            </div>
            </div>
            <div wire:ignore wire:key="{{ time(). 'noMorePost' }}" class="col-lg-12" id="noMorePost"  style="display: none; height: 100px; padding: 20px">
                <h3 class="text-center">No more posts :)</h3>
            </div>
            @endif
</div>`

**Reply by denizozturk** [reply]
What part of these codes do you want to be updated? where do you want not to update? The "wire:ignore.self" reference is often seen in modal examples, I might be able to help if I can figure out how to use it.

**Reply by mansoorkochy** [reply]
Thanks, Deniz, and I appreciate it, the problem is whenever an event inside `partials.livewire-post` component fired and something updated inside that component the element with id `postsBottom` lost its style, actually, I hide that element according to some condition but after updating the `'partials.livewire-post` livewire component it lost its style
